packages/dashboard/src/features/dashboard/hooks/useAdvisor.ts

```
@@ -66,27 +67,30 @@ const ADVISOR_COUNT_PAGE_SIZE = 100;
 export function useAdvisorCategoryCounts() {
   const host = useDashboardHost();
   const fetcher = host.onRequestAdvisorIssues;
+  const isEnabled = !!fetcher;
+
   return useQuery<AdvisorCategorySeverityMatrix, Error>({
     queryKey: ADVISOR_QUERY_KEYS.categoryCounts,
     queryFn: async () => {
-      if (!fetcher) {
-        return emptyMatrix();
-      }
+      if (!fetcher) return emptyMatrix();
```

ESLint

[error] Expected { after 'if' condition.

(curly)